### DISCUSSION TITLE
All data downloaded = 0 data (across multiple sites) any suggestions on where to start troubleshooting?

### DISCUSSION THREAD

**Discussion by Aston77** [AAndyProgram/SCrawler#125] [Q&A] [answered]
The program finds the users on twitter, IG & channels on Reddit, but F6 (Download All) results in no data being found and there are no thumbnails or images associated with the different users.

I believe all of the authorization information is properly in place for the different sites, but the way the program works I don't really know how/where to find any information about where the disconnect might be happening.

Any suggestions?

**Comment by AAndyProgram** [comment, marked as answer]
Even Reddit? This is impossible. Reddit can be parsed without cookies at all. Cookies are only needed by Reddit to parse saved posts.
I believe you have configured something wrong. Sorry.

If you still need help, please provide more information. Profile URLs, at least.

**Reply by Aston77** [reply]
I got it sorted out. I didn't quite understand the process for adding the cookies for the sites.

Thanks for getting back to me so fast.

BTW - It might just be me but I noticed that a lot of your hyperlinks in your guides don't seem to jump correctly. I tried it with both Edge & Firefox on two different computers and they don't seem to be responding like they do on other repos. Again....could just be on my end but I thought I would mention it. 🤷‍♂️

**Reply by AAndyProgram** [reply]
>It might just be me but I noticed that a lot of your hyperlinks in your guides don't seem to jump correctly.

Got it. I see. This is because I changed the default editor for md files. It needs an ASCII space separator instead of GitHub's "-". Sorry for that. I'll fix it soon.

**Reply by AAndyProgram** [reply]
Links fixed. Thank you for reporting this.

**Reply by Aston77** [reply]
> Links fixed. Thank you for reporting this.

Definitely makes things easier to use 👍.

**Reply by AAndyProgram** [reply]
😊
